src/cljs/velho_ds/pages/main.cljs

```diff
@@ -920,4 +904,83 @@
+                 {:name "content"
+                  :desc "components"
+                  :example "[areas/info\n[icons/type-icon-circle {:color color/color-pacific}]\n[fields/keyvalue {:label \"Type\"\n:content \"Project\"}]\n[fields/keyvalue {:label \"Schedule\"\n:content \"29.09.2017 - 01.01.2021\"}]\n[fields/keyvalue {:label \"State\"\n:content \"In Progress\"}]]"}]]
+
+   [:h2.rds-header2 "Content-header-default"]
+   [:p "Default structure of content-header. Parameters can be used to change values, but the styles are strict."]
+   ($-> [headings/content-header-default {:breadcrumb {:label "X-Files"
+                                                       :child {:label "Animals"
+                                                               :child {:label "Flying squirrel investigation"
+                                                                       :child nil}}}
+                                          :footnote "Related Organization, Another Organization"
+                                          :meta [{:content "Item"
+                                                  :on-click-fn #(println %)}
+                                                 {:content "Another Item"}
+                                                 {:content "Last Item"}]
+                                          :navigation [{:icon "info"
+                                                        :label "Info"
+                                                        :active true
+                                                        :on-click-fn #(println "Default icon-link clicked")}
+                                                       {:icon "insert_drive_file"
+                                                        :label "Documents"
+                                                        :on-click-fn #(println "Default icon-link clicked")}]
+                                          :theme-color color/color-pacific
+                                          :info-icon [icons/type-icon-circle]
+                                          :info-keyvalues [{:label "Type"
+                                                            :content "Project"}
+                                                           {:label "Schedule"
+                                                            :content "29.09.2017 - 01.01.2021"}
+                                                           {:label "State"
+                                                            :content "In Progress"}]}])
+   [props-table [{:name "breadcrumb"
+                  :desc "map"
+                  :example " {:breadcrumb {:label \"X-Files\"\n                                                       :child {:label \"Animals\"\n                                                               :child {:label \"Flying squirrel investigation\"\n                                                                       :child nil}}}}"}
```

Comment:
There is something wrong with the indents

Reply:
Fixed